src/psyclone/dynamo0p3.py

```diff
@@ -6792,20 +7048,13 @@ def _lower_bound_fortran(self):
         return mesh_obj_name + "%get_last_" + prev_space_name + "_cell(" \
             + prev_space_index_str + ")+1"
 
-    def _upper_bound_fortran(self):
-        ''' Create the associated fortran code for the type of upper bound
-
-        :return: Fortran code for the upper bound of this loop
+    @property
+    def _mesh_name(self):
+        '''
+        :returns: the name of the mesh variable from which to get the bounds \
```

Comment:
You use sometimes use `return` and sometimes `returns` in your modifications.

Reply:
I prefer `returns`. I'll try to stick to that.

Reply:
I was going to go through this file and check all my modifications but github isn't managing to render the diff at the moment (in either Firefox or Chrome).